R/RtGam.R

```diff
@@ -37,6 +37,10 @@
 #'   [mgcv::bam()] converges more quickly but introduces some additional
 #'   numerical error. Note that the `bam` backend uses the `discrete = TRUE`
 #'   option for an additional speedup. See [mgcv::bam()] for more information.
+#' @param warn_for_diagnostic_failure Should warnings be issued for
```

Comment:
another option (for its own issue) is to have a package wide verbose mode you control via `options()`.

Reply:
#28